core/cautils/display_test.go

```diff
@@ -30,3 +33,434 @@ func TestStartSpinner(t *testing.T) {
 		})
 	}
 }
+
+func TestFailureDisplay(t *testing.T) {
+	tests := []struct {
+		text string
+		want string
+	}{
+		{
+			text: "Test",
+			want: "Test",
+			// want: "\033[1;31mTest\033[0m",
```

Comment:
why did you drop the colors in all similar tests?

Reply:
I should've highlighted this point in the code itself. The output remains colourless all the time. At first, I thought that there was a bug in the code being used to capture the output. But, I tried directly printing using fmt and it was able to capture all the colors precisely. Which indicates that the outputs of all the display functions are colourless.

Reply:
ok, so pls remove these comments

Reply:
Done